appdev.Dockerfile

```diff
@@ -136,12 +136,6 @@ RUN /bin/bash -l -c "sudo apt update && sudo apt install -y graphviz=2.42.2-3bui
 # Install fuser (bin/server) and expect (web_git)
 RUN sudo apt install -y libpq-dev psmisc lsof expect
 
-# Install parity
-RUN wget -qO - https://apt.thoughtbot.com/thoughtbot.gpg.key | sudo apt-key add - \
-    && echo "deb http://apt.thoughtbot.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/thoughtbot.list \
-    && sudo apt-get update \
-    && sudo apt-get -y install parity=3.5.0-2
```

Comment:
Turns out those two thoughtbot links are dead now, which caused the `docker build` to fail. I was looking at the [parity gem](https://github.com/thoughtbot/parity), and it doesn't seem like something we need anymore? We're moving entirely to Render (and Fly), so do we need this gem or is it vestigial?

Removing it seemed shave off 0.02 GB, which is surprisingly large and gives me hope for reducing the size by quite a bit via other vestigial removals here (e.g. Heroku CLI in another commit).